tests/sentry/api/endpoints/test_group_event_details.py

```diff
@@ -143,14 +143,17 @@ def test_collapse_full_release(self):
 class GroupEventDetailsHelpfulEndpointTest(
     GroupEventDetailsEndpointTestBase, APITestCase, SnubaTestCase, OccurrenceTestMixin
 ):
-    def test_get_helpful_feature_off(self):
-        url = f"/api/0/issues/{self.event_a.group.id}/events/helpful/"
+    def test_get_recommended_feature_off(self):
```

Comment:
I don't think it's necessary to change the tests here. Since recommended is just an alias of helpful it's not super important to test both code paths (since they should be the same). When we remove `helpful` we can change these all to be `/recommended`.